docs/projects/version-control/converting/migrating-variables.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Migrating variables to Git
+description: Variables needs to be manually migrated to Git for existing projects
+position: 110
+hideInThisSection: true
+---
+
+Support for storing non-senstive variables in Git will be available to everyone in an upcoming version of Octopus. All newly configured Git projects will automatically have variables migrated when converting the project to Git - for existing Git projects, you will need to migrate these manually.
```

Comment:
If we know the version that will support it, we should include that to avoid ambiguity (in my opinion)

```suggestion
Support for storing non-sensitive variables in Git will be available to everyone in Octopus **2022.3**. All newly configured Git projects will automatically have variables migrated when converting the project to Git. For existing Git projects, you will need to migrate these manually.
```

Reply:
My plan with this is to be intentionally vague (for now), and put in an update with the specific versions once we've got the toggle enabled by default for everyone. I don't want to just put `2022.3` in there, because there are a bunch of cloud `2022.3` versions that do not support Git variables by default and may cause confusion.

Once we've got it locked down to the specific `2022.3` and `2022.4` versions that have it, I plan to update this.

Do we have a typical approach for things like this? We want to have the documentation out before we enable the feature for everyone, but we don't know the specific version until we do.

Reply:
That makes sense. Being vague is fine :smile:. There have been two approaches in the past:

1. Specify the exact version e.g. 2022.3.12345 it's available from
2. Specifying wording that doesn't commit to a version and then adding in the overall version e.g. 2022.3 when it's GA to both Cloud and Server (on-prem)

Both work :+1: